aiven/resource_vpc_peering_connection.go

```diff
@@ -378,7 +467,6 @@ func (w *VPCPeeringBuildWaiter) Conf(timeout time.Duration) *resource.StateChang
 			"ACTIVE",
 			"REJECTED_BY_PEER",
 			"PENDING_PEER",
-			"INVALID_SPECIFICATION",
```

Comment:
Why was this removed? This is usually a terminal state and there's no point waiting more if the peering connection goes into this state.

Reply:
Indeed, added it back